deploy/eck-stack/templates/tests/elasticsearch_test.yaml

```diff
@@ -4,7 +4,7 @@ templates:
 tests:
   - it: should render quickstart properly
     set:
-      eck-elasticsearch.version: 8.2.0
+      eck-elasticsearch.version: 8.15.0-SNAPSHOT
```

Comment:
Seems like this has not been updated in a while, so I think this change makes sense?